src/rust/ensogl/lib/core/src/display/scene.rs

```diff
@@ -328,7 +328,7 @@ impl Mouse {
         let position        = variables.add_or_panic("mouse_position",Vector2::new(0,0));
         let hover_ids       = variables.add_or_panic("mouse_hover_ids",target.to_internal(&logger));
         let target          = Rc::new(Cell::new(target));
-        let body            = web::dom::WithKnownShape::new(&web::document().body().unwrap());
```

Comment:
The name of the dom to embed EnsoGL is passed as an argument to scene or world constructor (cannot check now I’m on phone). This will not always be named “root” and hard coding the name I’m several places is rather a bad design. Please refactor it out so the name would not be hardcoded and will be passed from Scene instead.